### ISSUE TITLE
Tried it and updated it ..... a lot

### ISSUE THREAD

**Issue by davesmeghead** [wwolkers/pyvisonic#1]
Hi wwolkers,
I've tried your script and have run in to some problems, StartTimer is not defined and I get exceptions such as 'VisonicProtocol' object has no attribute 'PowerMaster'. Also you need "self." in front of the calls to displayzonebin.

Quite some time ago I too tried to get a python script working with my Powermax Pro and at the time I was pretty busy with other things and it didn't get progressed to completion. I've just tried to merge my code with yours this morning and I know you'll think I'm being pushy but I'm not. Just take a look and if there's anything useful then please feel free to use it, perhaps we could get it working together.
I have a TCP connection to my alarm and so I've also added a TCP comms function.
[pyvisonic.zip](https://github.com/wwolkers/pyvisonic/files/1998480/pyvisonic.zip)

Best Regards
David

**Comment by wwolkers** [comment]
OMG!
I recognize parts of it for sure, but you did rebuild a lot!

I love it! yeah let's see how far we can get with this thing. Feel like submitting a pull request and I'll put it in, no problem.

Also I noticed the dutch translation... fellow dutchie? :)

**Comment by davesmeghead** [comment]
OK, so I've followed the instructions on the internet and managed to create a fork and a pull request.
I did rebuild a lot but as I said I just merged what I had in to your structure, which was a lot better than mine!
I'm afraid that I'm not Dutch, sorry, I'm in England. The Dutch translations were copied from the Vera Lua plugin and converted to python.
Let me know how you get on, I'll keep plugging away in the background too.
